test/ts-scenario/steps/lib/deprecatedSDK.ts

```diff
@@ -225,23 +231,24 @@ export async function sdk_chaincode_instantiate(ccName: string, ccType: 'golang'
 		const deployId: string = proposalRequest.txId.getTransactionID();
 
 		const eventPromises: Promise<any>[] = [];
-		eventPromises.push(channel.sendTransaction(request as Client.TransactionRequest));
+		eventPromises.push(channel.sendTransaction(request));
 		eventHubs.forEach((eh: Client.ChannelEventHub) => {
 			const txPromise: Promise<any> = new Promise((resolve: any, reject: any): any => {
+				// eslint-disable-next-line @typescript-eslint/no-implied-eval
```

Comment:
I think `setTimeout(() => reject(), 300000)` would fix this. Perhaps it should be rejected with something like `new Error('timeout')` as the argument